crowbar_framework/app/models/tempest_service.rb

```diff
@@ -24,6 +24,18 @@ def initialize(thelogger)
     @logger = thelogger
   end
 
+  class << self
+    def role_constraints
+      {
+        "tempest" => {
+          "exclude_platform" => {
+            "suse" => "12.0"
+          }
+        }
```

Comment:
Since we're creating the constraints, can you add the other ones? I'd say that only one node can have the role.